platformio.ini

```diff
@@ -68,6 +68,21 @@ extra_scripts = tools/platformio_cfg_gitversion.py
 # To disable size optimization
 #build_unflags = -Os
 
+[env:host-teensy36]
+src_filter = +<common/*>,+<host/*>
+build_flags =
+    ${common.build_flags} -Isrc/common -Isrc/host
+    -DSERIAL1_RX_BUFFER_SIZE=512 -DSERIAL1_TX_BUFFER_SIZE=512
+    -DSERIAL2_RX_BUFFER_SIZE=256 -DSERIAL2_TX_BUFFER_SIZE=256
+    -DSERIAL3_RX_BUFFER_SIZE=256 -DSERIAL3_TX_BUFFER_SIZE=256
+platform = teensy
+framework = arduino
+board = teensy36
+lib_deps =
+  ${common.lib_deps_common}
+  ${common.lib_deps_teensy}
+extra_scripts = tools/platformio_cfg_gitversion.py
```

Comment:
I think you may need to add `lib_ignore = ${common.incompatible_libs_teensy}` when you merge. Not sure ... Please give it a try. I have added it for `env:host`.

Reply:
I am afraid I did some mistake on the last merge of your updates, because
`[env:host-teensy36] [env:program-esp]  [env:program-esp-teensy36]`
are already updated in your uptodate master branch.
It was doubled in the PR by mistake.

did a
`git checkout upstream/master platformio.ini`
now